### PR TITLE
Fixes to the documentation

### PR DESCRIPTION
**Pull request by gsketefian**
## DESCRIPTION OF CHANGES: 
1) ConfigNewPlatform.rst, ConfigParameters.rst, and ConfigWorkflow.rst:  Minor changes (capitalize "rocoto" where appropriate).
2) InputOutputFiles.rst:  Add link to GST files on AWS.
3) Graphics.rst:  Requested fixes and other improvements.
4) RocotoInfo.rst:  Convert subsections to sections (previously there was only one section with several subsections).
5) WE2Etests.rst:  Add more details, clarifications, and fixes.

## TESTS CONDUCTED: 
None.

### CODE REVIEW

**Comment by gsketefian** [comment]
There is one more fix that I think @jwolff-ncar would like in the graphics chapter, which is how to specify CDATE.  Using 

`export CDATE=${DATE_FIRST_CYCL}${CYCL_HRS}
`

(see end of that chapter) happens to work if you're running just one cycle, but in general it won't.  I don't have time to fix this today.   Up to you all if you want to merge this today or wait until tomorrow afternoon for a fix (or someone else just make a new PR for that fix).

File: docs/UsersGuide/source/Graphics.rst
Comment (jwolff-ncar) [inline docs/UsersGuide/source/Graphics.rst:116]: This should have a "." at the end rather than a ","

**Review by jwolff-ncar — COMMENTED** [review]
I would prefer to leave CDATE defined as is in the batch script. These scripts will not work for multiple cycles as is so no need to make it fancier at this point. (Plus, NO MORE code changes!) If someone runs multiple cycles they will need to hard code the CDATE for the cycle they want to visualize. What we should do instead is add the following (in bold below) to the documentation at the end of the graphics chapter:

"In addition, the following variables can be modified in the batch scripts depending on your needs (for example, **if you have run multiple cycles and you want to plot output from a cycle other than the very first one, CDATE may be set to YYYYMMDDHH;** if you want to plot hourly forecast output, FCST_INC should be set to 1; if you just want to plot a subset of your model output, you can set FCST_START, FCST_END, and FCST_INC accordingly):

File: docs/UsersGuide/source/InputOutputFiles.rst
Comment (jwolff-ncar) [inline docs/UsersGuide/source/InputOutputFiles.rst:331]: These files are also linked at the end of section 7.3.2 so we  are well covered.

Comment (jwolff-ncar) [reply docs/UsersGuide/source/InputOutputFiles.rst:331]: One more request. Please add the following to the graphics chapter (in  bold below). 

"The **full set  of** Cartopy shape files can be downloaded at https://www.naturalearthdata.com/downloads/. **For ease, the small subset of files required for these Python scripts can be obtained from the EMC ftp data repository **(linked to: https://ftp.emc.ncep.noaa.gov/EIB/UFS/SRW/v1p0/natural_earth/natural_earth_ufs-srw-release-v1.0.0.tar.gz)** or from AWS cloud storage **(linked  to https://ufs-data.s3.amazonaws.com/public_release/ufs-srweather-app-v1.0.0/natural_earth/natural_earth_ufs-srw-release-v1.0.0.tar.gz)**.** The medium scale..."

**Comment by jwolff-ncar** [comment]
One more request. Please add the following to the graphics chapter (in bold below).

"... 
The **full set of** Cartopy shape files can be downloaded at https://www.naturalearthdata.com/downloads/. **For ease, the small subset of files required for these Python scripts can be obtained from the EMC ftp data repository** (linked to: https://ftp.emc.ncep.noaa.gov/EIB/UFS/SRW/v1p0/natural_earth/natural_earth_ufs-srw-release-v1.0.0.tar.gz) **or from AWS cloud storage** (linked to https://ufs-data.s3.amazonaws.com/public_release/ufs-srweather-app-v1.0.0/natural_earth/natural_earth_ufs-srw-release-v1.0.0.tar.gz)**.** The medium scale..."

**Comment by jwolff-ncar** [comment]
I have not assessed the WE2E test chapter changes. I am Hopeful someone else can read through that  and provide comments if needed.

**Comment by climbfuji** [comment]
> I have not assessed the WE2E test chapter changes. I am Hopeful someone else can read through that and provide comments if needed.

My plan is to read through the entire documentation on the weekend and note whatever differs from the testing I have done so far and will do tomorrow.

File: docs/UsersGuide/source/Graphics.rst
Comment (gsketefian) [reply docs/UsersGuide/source/Graphics.rst]: Done.

**Comment by gsketefian** [comment]
I've addressed all the requested changes.  Just need an approval to merge.

**Review by JeffBeck-NOAA — APPROVED** [review]

**Comment by jwolff-ncar** [comment]
Content looks good. I did not check the formatting so that should be done once it has been merged if Gerard was not able to get that working on his machine.
Thanks!

**Review by jwolff-ncar — APPROVED** [review]

**Comment by gsketefian** [comment]
> Content looks good. I did not check the formatting so that should be done once it has been merged if Gerard was not able to get that working on his machine.
> Thanks!

Fortunately, I was able to get sphinx to work on my laptop and checked the content.  Looked good, but always appreciate someone else looking it over.  Will merge now.

**Pull request merged**
